src/cms/data/functions/createSearchIndexes.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+/* Creates a generated searchtext column on protected_areas, if it doesn't already exist */
+module.exports = async () => {
+  try {
+    const knex = strapi.connections[strapi.config.database.defaultConnection];
+
+    const hasSearchColumn = await knex.schema.hasColumn(
+      "protected_areas",
+      "search_text"
+    );
+
+    if (!hasSearchColumn) {
```

Comment:
@BcGovNeal this seemed like a reasonably safe way to handle index & generated column creation. Let me know if you have concerns or can see ways to improve it.

Reply:
I think this is fine as long as we don't introduce other ways of modifying the same schema